src/fides/api/util/saas_util.py

```diff
@@ -435,6 +435,19 @@ def get_identity(privacy_request: Optional[PrivacyRequest]) -> Optional[str]:
     return identities[0] if identities else None
 
 
+def get_identities(privacy_request: Optional[PrivacyRequest]) -> List[str]:
+    """
+    Returns a list of cached identity names for the provided privacy request.
+    """
+
+    if not privacy_request:
+        return []
+
+    cached_identity_data: Dict[str, Any] = privacy_request.get_cached_identity_data()
+    identities = [k for k, v in cached_identity_data.items() if v]
+    return identities
+
+
```

Comment:
nit: i think i'd rather see this return a `Set[str]`, since you'll be looking for membership (`if param_value.identity in get_identities(self.privacy_request)`), and you don't care about the order. of course the number of elements here is never going to be large enough that it'll matter for performance, but this is just my pet-peeve of appropriate data structures coming into play 😛 . in this case, i think it's a pretty simple switch and it just feels a bit cleaner to me! 

```suggestion
def get_identities(privacy_request: Optional[PrivacyRequest]) -> Set[str]:
    """
    Returns a set of cached identity names for the provided privacy request.
    """

    if not privacy_request:
        return []

    cached_identity_data: Dict[str, Any] = privacy_request.get_cached_identity_data()
    identities = {k for k, v in cached_identity_data.items() if v}
    return identities

```

Reply:
You got it! Kelsey always says "you've never really said anything until you've said it three times". You might have to tell me once more and then it will definitely sink in 😆